### DISCUSSION TITLE
[Exercise-3]: Write in a few words about navigating through a schema object (for e.x. table) using SQL developer

### DISCUSSION THREAD

**Discussion by acrlakshman** [exaslate-courses/oracle-sql-plsql-b3#4] [General]
Ref.: https://www.youtube.com/watch?v=S0XnhFOmCJ0&list=PLF6_5RI6ENIlDi3knzcQGUiFc5-WdA_1W&index=1&t=2064s

**Comment by srideviadapa0511** [comment]
we can access any table we can follow a short cut by clicking on the right side and click open declaration. And there we can see table with their values.On top of the page we can see few tabs for executing program in sql developer we can select sql then will get to see all the code written using hr.user.Below i attached screenshot of table opened in sql developer.

**Reply by acrlakshman** [reply]
@srideviadapa0511 excellent for responding using your experience.

**Comment by srideviadapa0511** [comment]
![hr schema](https://github.com/exaslate-courses/oracle-sql-plsql-b3/assets/145452411/e1ce8c38-40cb-455a-953a-0a1529fa4bb0)

**Comment by Tharun1596** [comment]
we can navigate any table by right click and open declaration on sql developer, then we can see the tables with values.
On the top we can see some tabs for executing sql program.
click on sql we can see the codes written in hr.user.

**Comment by YaminiYadavv** [comment]
To navigate a schema object to the values or code, we can right click on the table that we want to get details and select open declaration than we get a popup with file which we mentioned, and the values can be shown on top bar icons where we have to SQL from those tabs.
Once we select SQL, we get the DDL, DML statements which are used to Create a table and Insert Values into the table followed by some Query languages.

**Reply by acrlakshman** [reply]
@YaminiYadavv Nice explanation. Quick correction, not `anywhere in the SQL developer`, you have to keep cursor on the table that you want to get details of.
